mypyc/test-data/run-lists.test

```diff
@@ -125,7 +125,7 @@ def test_insert() -> None:
             l.insert(long_int, 5)
         except Exception as e:
             assert type(e).__name__ == 'OverflowError'
-            assert str(e) == 'Python int too large to convert to C ssize_t'
+            assert str(e) == 'insert index too large for list'
```

Comment:
As discussed above, after thinking about this more carefully, it looks better to preserve the old message. Add comment here saying that the message is used by CPython.